src/web/app/pages-instructor/student-list/student-list.component.ts

```diff
@@ -18,29 +18,72 @@ import { StudentListSectionData, StudentListStudentData } from './student-list-s
   templateUrl: './student-list.component.html',
   styleUrls: ['./student-list.component.scss'],
 })
-export class StudentListComponent implements OnInit {
+export class StudentListComponent implements OnInit, DoCheck {
   @Input() courseId: string = '';
-  @Input() sections: StudentListSectionData[] = [];
   @Input() useGrayHeading: boolean = true;
   @Input() listOfStudentsToHide: string[] = [];
   @Input() isHideTableHead: boolean = false;
   @Input() enableRemindButton: boolean = false;
+  @Input() sections: StudentListSectionData[] = [];
+
+  students: StudentListStudentData[] = [];
```

Comment:
Add a comment for this (to answer why both `sections` and `students` are here)?

Reply:
Updated